File: 01_140407/test0.php
<?php
function test($i, $j) {
    return $i * $j;
}

function main() {
    $i = 3;
    $j = 2;
    $k = test($i, $j);
    if ($k > 5) echo ">5\n";
    else echo "<=5\n";
}

main();

File: 01_140407/test0.rb
def test(i, j)
    return i*j
end

def main()
    i = 3
    j = 2
    k = test(3, 2)
    if k > 5
        print(">5\n")
    else
        print("<=5\n")
    end
end

main()
